systemtests/src/test/java/io/enmasse/systemtest/isolated/certs/CertProviderTest.java

```diff
@@ -4,17 +4,7 @@
  */
 package io.enmasse.systemtest.isolated.certs;
 
-import io.enmasse.address.model.Address;
```

Comment:
please stop using `*` import :)

Reply:
Gah, i need to setup my ide config again :)

Reply:
:D :D yup